src/naksu/ui/progress/dialog.go

```diff
@@ -0,0 +1,70 @@
+package progress
+
+import (
+	"github.com/andlabs/ui"
+	"naksu/xlate"
+	"strconv"
+)
```

Comment:
Go-koodauskonvention mukaan importit tulevat järjestykeen 1) stdlib, 2) kolmannen osapuolen kirjastot 3) tämän projektin muut moduulit, ja nämä kolme ryhmää pitää erottaa tyhjällä rivillä:

```suggestion
import (
	"strconv"

	"github.com/andlabs/ui"

	"naksu/xlate"
)
```